### PR TITLE
Add OmenDisputeResolution Scheme with SummaryCustom UI

### PR DESCRIPTION
**Pull request by nicoelzer**
This commit provides a GenericScheme to integrate a DAOstack DAO with realit.io as arbitrator to resolve disputes(https://realitio.github.io/docs/html/arbitrators.html)

This PR is intended to test the Scheme on Rinkeby.

### CODE REVIEW

**Comment by dkent600** [comment]
If the purpose is to resolve disputes around rewarding of ContributionRewards, I recommend an architecture using ContributeRewardEx such as enabled the Competition contract to work as it does.

**Comment by pimato** [comment]
@dkent600 this generic scheme has nothing to do with the ContributionRewards . @nicoelzer already mentioned that this generic scheme is specifically for disputing an arbitration made by the realit.io protocol.

**Comment by dkent600** [comment]
> @dkent600 this generic scheme has nothing to do with the ContributionRewards . @nicoelzer already mentioned that this generic scheme is specifically for disputing an arbitration made by the realit.io protocol.

Cool, but just to let you know why I mentioned this:  Arbitration, dispute resolution, is totally applicable to the use case where there is a dispute about whether contribution rewards should be rewarded.  Perhaps it needs to be confirmed that the promised project has been completed.  Etc...  Can realit.io not be used in such a situation?

File: test/integration/proposal-genericSchemeOmenDisputeResolution.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*import * as uuid from "uuid";
```
Comment (orenyodfat) [inline test/integration/proposal-genericSchemeOmenDisputeResolution.ts:1]: why this test is comment out  ?

**Comment by orenyodfat** [comment]
@nicoelzer looks good. 
please note that we currently encounter issue with rinkeby subgraph . I guess it will take few days till it will be solved.
can you try to switch to kovan ?

**Comment by nicoelzer** [comment]
> @nicoelzer looks good.
> please note that we currently encounter issue with rinkeby subgraph . I guess it will take few days till it will be solved.
> if this is a burning feature , it might be worth to deploy it on kovan . we then can have alchemy staging app on kovan it can be reviewed.

@orenyodfat thanks! Sure thing, anything to prepare for deploying on rinkeby besides adding the kovan address within the SchemeRegistry json? Whats the url of Alchemy Kovan Dapp?

File: src/components/Proposal/ProposalSummary/ProposalSummaryOmenDisputeResolution.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+    const proposalSummaryClass = classNames({
+      [css.detailView]: detailView,
+      [css.transactionModal]: transactionModal,
+      [css.proposalSummary]: true,
+      [css.withDetails]: true,
+    });
+
+    switch (action.id) {
+      case "setMetaData":
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalSummary/ProposalSummaryOmenDisputeResolution.tsx:39]: all of these look almost identical with only very minor changes in one part, maybe just use the case statement to change that one part so you dont have so much duplicate code?

**Pull request closed without merging**